lib/middleware/token.js

```diff
@@ -51,6 +51,8 @@ module.exports = function token(server, options) {
   if (!server) { throw new TypeError('oauth2orize.token middleware requires a server argument'); }
   
   return function token(req, res, next) {
+    if (!req.body) { return next(new Error('OAuth2orize requires body parsing. Did you forget app.use(express.bodyParser())?')); }
```

Comment:
I have only take error message in other file in this project. Effectivly,
errors messages and express example are outdated.
Le 16 févr. 2015 17:51, "Christian Tellnes" <notifications@github.com> a
écrit :

> In lib/middleware/token.js
> <https://github.com/jaredhanson/oauth2orize/pull/116#discussion_r24762220>
> :
>
> > @@ -51,6 +51,8 @@ module.exports = function token(server, options) {
> >    if (!server) { throw new TypeError('oauth2orize.token middleware requires a server argument'); }
> >
> >    return function token(req, res, next) {
> > +    if (!req.body) { return next(new Error('OAuth2orize requires body parsing. Did you forget app.use(express.bodyParser())?')); }
>
> express does not bundle bodyParser any more. See
> http://expressjs.com/4x/api.html#req.body for details.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jaredhanson/oauth2orize/pull/116/files#r24762220>.
>